projects/assets/src/translations/translation-chunks-config.ts

```diff
@@ -1,6 +1,4 @@
-interface TranslationChunksConfig {
```

Comment:
This interface was redundant in the assets lib on purpose - not to have a dependency between `@spartacus/assets` and `@spartacus/core`